RoutingStrategies.xaml

```diff
@@ -0,0 +1,147 @@
+                but it was nice to get practice on tabbed pages and work with new people.
+-->
+<TabbedPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+            xmlns:local="clr-namespace:Lab6_Starter"
+            xmlns:maps="clr-namespace:Microsoft.Maui.Controls.Maps;assembly=Microsoft.Maui.Controls.Maps"
+            x:Class="Lab6_Starter.RoutingStrategies"
+            Title="Routing Strategies"
+            xmlns:android="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;assembly=Microsoft.Maui.Controls"
+            android:TabbedPage.ToolbarPlacement="Bottom">
+
+    <ContentPage Title="Airports"
+                 IconImageSource="rectangle_list_48.png"
+                 xmlns:maps="http://schemas.microsoft.com/dotnet/2021/maui/maps">
+
+        <VerticalStackLayout Padding="20">
+
+            <!-- Starting Airport (Carissa)-->
+            <HorizontalStackLayout>
+                <Label Text="Starting Airport:"
+                       VerticalOptions="Center"
+                       Margin="0,20,10,20"/>
+                <Frame BorderColor="Gray"
+                       CornerRadius="5"
+                       Padding="0"
+                       HasShadow="True">
+                    <Entry x:Name="StartingAirportPicker"
+                           Text="KATW"
+                           WidthRequest="55"
+                           HeightRequest="40"/>
+                </Frame>
+            </HorizontalStackLayout>
+
+
+            <!-- Max Distance (Pachia)-->
+            <HorizontalStackLayout Padding="0,10">
+                <Label Text="Max Distance:"
+                       VerticalOptions="Center"
+                       Margin="0,10,10,20"/>
+                <Frame BorderColor="Gray"
+                       CornerRadius="5"
+                       Padding="5"
+                       HasShadow="True">
+                    <Entry x:Name="MaxDistanceEntry"
+                           Text="60"
+                           Keyboard="Numeric"
+                           WidthRequest="30"
+                           HeightRequest="40"/>
+                </Frame>
+                <Label Text="  nm"
+                       VerticalOptions="Center"/>
+            </HorizontalStackLayout>
+
+            <!-- Unvisited Toggle (Hannah) -->
+            <HorizontalStackLayout>
+                <Label Text="Unvisited:"
+                       VerticalOptions="Center"/>
+                <Switch x:Name="UnvisitedSwitch"
+                        IsToggled="False"/>
+            </HorizontalStackLayout>
+
+            <!-- Suggested Routes with Frame (Hannah)-->
+            <Label Text="Suggested Routes"
+                   FontAttributes="Bold"
+                   FontSize="Medium"
+                   Margin="0,20,0,0"/>
+            <ListView x:Name="RoutesListView"
```

Comment:
I didn't realize you were using a ListView, those are somewhat deprecated. Can you switch this to a CollecitonView? Can you left-justify the text? It's centered right now, which looks odd.